editors-definitions/che-idea-server-latest.yaml

```diff
@@ -13,13 +13,14 @@
 schemaVersion: 2.2.2
 metadata:
   name: che-idea-server
-  displayName: IntelliJ IDEA Ultimate (desktop)
+  displayName: Jet Brains IntelliJ IDEA Ultimate (desktop)
   description: JetBrains IntelliJ IDEA Ultimate dev server for Eclipse Che - latest
   tags:
     - Tech-Preview
   attributes:
     publisher: che-incubator
-    version: latest
+    version: latest (2024.2.3)
```

Comment:
looks nice on UI, but I have some concerns about the factory flow e.g. `che-editor=publisher/editor-name/version`

Reply:
the version is moved to `dispalyName`